doc/nrf/releases/release-notes-2.2.0.rst

```diff
@@ -0,0 +1,984 @@
+      This allows calling the :c:func:`emds_store` function from an interrupt context.
+    * The internal thread for storing the emergency data has been removed.
+      The emergency data is now stored by the :c:func:`emds_store` function.
+
+* :ref:`mod_dm` module:
+
+  * Added a window length configuration to be used runtime, when a new measurement request is added.
+
+  * Updated:
+
+    * The calculation of MPSL timeslot length has been improved by using the :ref:`nrfxlib:nrf_dm` functionality.
+    * Renamed the ``access_address`` field to ``rng_seed`` in the :c:struct:`dm_request` structure.
+
+* :ref:`app_event_manager`:
+
+  * Added :kconfig:option:`CONFIG_APP_EVENT_MANAGER_SHELL` Kconfig option.
+    The option can be used to disable Event Manager shell commands.
+
+* :ref:`nrf_profiler`:
+
+  * Added the :kconfig:option:`CONFIG_NRF_PROFILER_SHELL` Kconfig option.
+    The option can be used to disable the nRF Profiler shell commands.
+
+Common Application Framework (CAF)
+----------------------------------
+
+* :ref:`caf_ble_state`:
+
+  * Added :kconfig:option:`CONFIG_CAF_BLE_STATE_MAX_LOCAL_ID_BONDS`.
+    The option allows to specify the maximum number of allowed bonds in each Bluetooth local identity for a Bluetooth Peripheral.
+
+* :ref:`caf_sensor_manager`:
+
+  * Added:
+
+    * Dynamic control for the :ref:`sensor sample period <sensor_sample_period>`.
+    * Test for the sample.
+
+DFU libraries
+-------------
+* :ref:`lib_dfu_target` library:
+
+  * Added:
+
+    * New types :c:enum:`DFU_TARGET_IMAGE_TYPE_ANY_MODEM` and :c:enum:`DFU_TARGET_IMAGE_TYPE_ANY_APPLICATION`.
+      This makes any supported modem update type acceptable when downloading.
+
+  * Updated:
+
+    * Calling the :c:func:`dfu_target_reset()` function clears all images that have already been downloaded into a target area.
+      This allows cancelling any update packages even if they are already marked to be updated.
+
+MPSL libraries
+--------------
+
+* Added :ref:`mpsl_lib`.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+* :ref:`west_sbom`:
+
+  * Updated the SPDX License List database to version 3.18.
+
+* :ref:`partition_manager`:
+
+  * Added:
+
+    * :kconfig:option:`CONFIG_PM_PARTITION_ALIGN_SETTINGS_STORAGE` Kconfig option to specify the alignment of the settings storage partition.
+    * P-GPS partition section to the :file:`ncs/nrf/subsys/partition_manager/Kconfig` file.
+    * Board-specific static Partition Manager configuration shared among build types.
+      The configuration can be defined in the board's directory.
+
+  * Updated the :file:`ncs/nrf/subsys/partition_manager/pm.yml.pgps` file to place P-GPS partition in external flash when so configured.
+
+  * Fixed:
+
+    * An issue with the driver and devicetree symbol for the external flash memory where the driver was sometimes NULL, even if the DT node was chosen.
```

Comment:
This can cause a somewhat cryptic error for mcuboot users, where they need to add a devicetree overlay to add a chosen `nordic,pm-ext-flash`. Maybe there should be a note here, or somewhere to do with mcuboot. Here is an example of this being fixed: https://github.com/nrfconnect/sdk-nrf/pull/9487/files.

@Damian-Nordic might have an opinion.

Reply:
@Damian-Nordic, could you suggest?

Reply:
Added this, @Damian-Nordic.